packages/block-library/src/post-comments/edit.js

```diff
@@ -78,7 +44,138 @@ export default function PostCommentsEdit( {
 			</BlockControls>
 
 			<div { ...blockProps }>
-				<PostCommentsDisplay postId={ postId } />
+				<Warning>
+					{ __(
+						'This is just a placeholder, not a real comment. The final styling may differ because it also depends on the current theme. For better compatibility with the Block Editor, please consider replacing this block with the "Comments Query Loop" block.'
+					) }
+				</Warning>
+
+				<h3>
+					{ __( 'One response to' ) } “{ postTitle }”
+				</h3>
+
+				<div className="navigation">
+					<div className="alignleft">
+						<a href="#top">« { __( 'Older Comments' ) }</a>
+					</div>
+					<div className="alignright">
+						<a href="#top">{ __( 'Newer Comments' ) } »</a>
+					</div>
+				</div>
+
+				<ol className="commentlist">
+					<li className="comment even thread-even depth-1">
+						<article className="comment-body">
+							<footer className="comment-meta">
+								<div className="comment-author vcard">
+									<img
+										alt=""
+										src="http://1.gravatar.com/avatar/d7a973c7dab26985da5f961be7b74480?s=32&d=mm&r=g"
```

Comment:
It should be theoretically safe because it's what WordPress is using on new installations, but I agree it'd be better to have something that is not hardcoded.

I'll use the default avatar exposed in the state 🙂

Reply:
Done in https://github.com/WordPress/gutenberg/pull/40484/commits/b3b2b6c824d6cb153ee3b0b17e0f7d9a05ee5d64.